Formula/q/qt@5.rb

```diff
@@ -31,7 +30,7 @@ class QtAT5 < Formula
 
   depends_on "node" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.12" => :build
+  depends_on "python@3.12" => :build # Python 3.12 fails with ModuleNotFoundError: No module named 'pipes'
```

Comment:
May do this in syntax-only follow up if CI passes.
```suggestion
  depends_on "python@3.12" => :build # Python 3.13 fails with ModuleNotFoundError: No module named 'pipes'
```